fsspec/registry.py

```diff
@@ -86,6 +86,7 @@ def register_implementation(name, cls, clobber=True, errtxt=None):
 # updated with register_implementation
 known_implementations = {
     "file": {"class": "fsspec.implementations.local.LocalFileSystem"},
+    "local": {"class": "fsspec.implementations.local.LocalFileSystem"},
```

Comment:
If you add this, you also need to add "local" to the FSs `protocols` attribute, otherwise passing "local://..." URLs will fail with some operations. Also, this is not now tested anywhere.